.projenrc.js

```diff
@@ -85,6 +85,18 @@ const installHelm = project.addTask('install-helm', {
 
 project.testTask.prependSpawn(installHelm);
 
+// test compatibility with constructs v10
+const compatTask = project.addTask('test:compat');
+compatTask.exec('cp package.json package.json.bak');
+compatTask.exec('yarn add --dev constructs@^10');
```

Comment:
I rather us to declare our build dependency ^10 and validate against ^3 instead of the other way around (^3 is "legacy").

Reply:
Good idea.